keyboards/monsgeek/m1/config.h

```diff
@@ -0,0 +1,89 @@
+/* Copyright (C) 2022 jonylee@hfd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Use 5 dynamic keymap layers */
+#define DYNAMIC_KEYMAP_LAYER_COUNT 6
+
+#define RGB_TRIGGER_ON_KEYDOWN
+/* Force NKRO on boot up regardless of the setting saved in the EEPROM (uncomment to enable it) */
+#define FORCE_NKRO
+
+/* encoder resolution */
+#define TAP_CODE_DELAY 20
```

Comment:
That is what i  commented here: 
https://github.com/qmk/qmk_firmware/pull/19799#issuecomment-1464883227